content/studies/sample.md

```diff
@@ -1,7 +1,9 @@
 ---
 title: Regenerative Phone - The Light II
 tags: Lightweight LCA, Consumer Education, Meaningful Offsets
-hero_image: ../public/studies/light-two/hero.jpg
+hero_image: /img/studies/light-two/hero.jpg
+hero_image_caption: This is a caption.
+type: Case Study
 ---
 # Investigating the climate impacts of a mobile phone. Building a novel checkout feature to invite customers into the sustainability conversation.
```

Comment:
adding a placeholder case study

Reply:
out of curiosity, where is /img/studies/light-two/hero.jpg coming from if it's not an asset in our repo?

Reply:
@bbqxaxiu `/img/studies/light-two/hero.jpg` is a saved asset in our repo — lmk if I misunderstood the question?

Reply:
Sure thing — you should be able to find it [here](https://github.com/sanctuarycomputer/seaborne-2.0/blob/main/public/img/studies/light-two/hero.jpg) - lmk in case of any questions!